server/pbenchinacan/container-build-sa.sh

```diff
@@ -44,17 +44,17 @@ fi
 container=$(buildah from --format=docker ${BASE_IMAGE})
 
 # Provide a few useful defaults for labels and the host name to use, and expose
-# port 80 for the Apache2 httpd process acting as a proxy.
+# port 8080 for the Nginx proxy server.
 buildah config \
     --label maintainer="Pbench Maintainers <pbench@googlegroups.com>" \
-    --port 80      `# pbench-server` \
+    --port 8080      `# pbench-server` \
```

Comment:
Given that we're exposing only one port, and given that it is described at line 47 (and, given that the port is actually attached to the proxy server and not the Pbench Server...), we should remove the `'# pbench-server'` from line 50.